pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -2512,6 +2509,17 @@ func parseOpCmds(getconfigCtx *getconfigContext,
 	return reboot, shutdown
 }
 
+// parseLocConfig() - assign LOC config only if URL is valid
+func parseLocConfig(getconfigCtx *getconfigContext,
+	config *zconfig.EdgeDevConfig) {
+	locConfig := config.GetLocConfig()
+	if locConfig != nil && len(locConfig.LocUrl) > 0 {
```

Comment:
Shouldn't the controller be able to clear getconfigCtx.locConfig by passing nil?

Reply:
There is a check for null.

Reply:
Yes, but when nil you don't update  getconfigCtx.locConfig to be nil, hence I don't see how the controller can disable it.